.ibm/pipelines/value_files/values-showcase_auth-providers.yaml

```diff
@@ -0,0 +1,239 @@
+global:
+  dynamic:
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-keycloak-backend-dynamic
+        disabled: true
+      - package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-org-dynamic
+        disabled: true
+      - package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-msgraph-dynamic
+        disabled: true
+      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-rbac
+        disabled: true
+upstream:
+  backstage:
+    appConfig:
+      signInPage: oidc
+      auth:
+        environment: production
+        session:
+          secret: superSecretSecret            
+        providers:
+          guest:
+            dangerouslyAllowOutsideDevelopment: true
+          microsoft:
+            development:
+              clientId: ${AUTH_PROVIDERS_AZURE_CLIENT_ID}
+              clientSecret: ${AUTH_PROVIDERS_AZURE_CLIENT_SECRET}
+              tenantId: ${AUTH_PROVIDERS_AZURE_TENANT_ID}
+              domainHint: ${AUTH_PROVIDERS_AZURE_TENANT_ID}
+            production:
+              clientId: ${AUTH_PROVIDERS_AZURE_CLIENT_ID}
+              clientSecret: ${AUTH_PROVIDERS_AZURE_CLIENT_SECRET}
+              tenantId: ${AUTH_PROVIDERS_AZURE_TENANT_ID}
+              domainHint: ${AUTH_PROVIDERS_AZURE_TENANT_ID}
+          github:
+            production:
+              clientSecret: ${AUTH_ORG_CLIENT_SECRET}
+              clientId: ${AUTH_ORG_CLIENT_ID}
+          oidc:
+            production:
+              metadataUrl: ${RHSSO76_METADATA_URL}
+              clientId: ${RHSSO76_CLIENT_ID}
+              clientSecret: ${RHSSO76_CLIENT_SECRET}
+              prompt: auto 
+              callbackUrl: ${RHSSO76_CALLBACK_URL}
+      integrations:
+        github:
+          - host: github.com
+            apps:
+              - appId: ${AUTH_ORG_APP_ID}
+                clientId: ${AUTH_ORG_CLIENT_ID}
+                clientSecret: ${AUTH_ORG_CLIENT_SECRET}
+                privateKey: ${AUTH_ORG1_PRIVATE_KEY}
+                webhookSecret: ${AUTH_ORG_WEBHOOK_SECRET}
+      catalog:
+        locations:
+          - type: url
+            target: https://github.com/albarbaro-org/some-test/blob/main/location.yaml
```

Comment:
can you please move this to janus-qe org, so in future if needed anyone can update.